cosmos/util/signal_handlers.py

```diff
@@ -92,17 +110,16 @@ def main():
                 workflow.run()
     """
 
-    def __init__(
-        self, workflow,
-        lethal_signals=frozenset([signal.SIGINT, signal.SIGTERM,
-                                  signal.SIGUSR2, signal.SIGXCPU]),
-        benign_signals=frozenset([signal.SIGUSR1]),
-        explanations={
-            signal.SIGUSR1: 'SGE is about to send a SIGSTOP, or, '
-                            'if a time limit has been exceeded, a SIGKILL',
-            signal.SIGUSR2: 'SGE is about to send a SIGKILL',
-            signal.SIGXCPU: 'SGE is about to send a SIGKILL, '
-                            'because a cpu resource limit has been exceeded'}):
+    def __init__(self, workflow,
+                 lethal_signals=frozenset([signal.SIGINT, signal.SIGTERM,
+                                           signal.SIGUSR2, signal.SIGXCPU]),
+                 benign_signals=frozenset([signal.SIGCONT, signal.SIGUSR1]),
+                 explanations={
```

Comment:
Lint flags this line, I could set `explanations=None` and then fill it in with a default value. That's probably the "most correct" way to handle all these parameters. Let me know if you'd like me to make that change. This text is green because I linted its whitespace. The only change here is adding signal.SIGCONT to the list of benign signals.